internal/pkg/template/workload.go

```diff
@@ -91,6 +91,12 @@ var (
 	osFamiliesForPV100 = []string{
 		OSWindowsServerFull, OSWindowsServerCore,
 	}
+
+	// Changing the suffix version number will force trigger the env controller for once.
+	// The suffix version number should be the latest environment template version at the time when a force trigger needs to happen.
+	// This is useful when we add a new output in the environment stack, and we need to reference
+	// the new output from the workload stack. See example: https://github.com/aws/copilot-cli/pull/3018
+	forceTriggerEnvController = "ForceTriggerEnvController-v1.7.0"
```

Comment:
Is the goal of this parameter to once and only once trigger the envcontroller to fetch outputs? Am I  misunderstanding something?


Is there a particular reason why we can't do, something like this:
```
{{- if eq .NLB.Listener.Protocol "TLS"}}
DependsOn: EnvControllerAction
{{- end }}
Properties:
 ...
{{- if eq .NLB.Listener.Protocol "TLS" }}
    Certificates:
    - CertificateArn:
         Fn::ImportValue: !Sub '${AppName}-${EnvName}-HTTPSCertArn'
```

Let me know if I'm missing something 🤔 I'd really love to be able to be able to remove this parameter if possible